internal/docker-run/action.yml

```diff
@@ -36,6 +36,13 @@ inputs:
 runs:
   using: 'composite'
   steps:
+    - name: Validate Docker image
+      if: ${{ github.event_name == 'pull_request_target' && contains(inputs.image, 'release') }}
```

Comment:
I think we could make this a little bit more specific.  The repository we want to protect is `us-docker.pkg.dev/protobuf-build/release-containers/*`